locales/en-US/pocket.properties

```diff
@@ -26,6 +26,8 @@ tagline = Save articles and videos from Firefox to view in Pocket on any device,
 taglinestory_one = Click the Pocket Button to save any article, video or page from Firefox.
 taglinestory_two = View in Pocket on any device, any time.
 tagssaved = Tags Added
+tos = By continuing, you agree to Pocket's <a href="https://getpocket.com/tos?s=ffi&t=tos&tv=panel_tryit">Terms of Service</a> and <a href="https://getpocket.com/privacy?s=ffi&t=privacypolicy&tv=panel_tryit">Privacy Policy</a>
```

Comment:
Is there any chance to move the actual URLs outside of the string, and use placeholders here? That way you're free to change URLs any time you want, without touching strings.